Example/ReusableDemo iOS/TableViewController.swift

```diff
@@ -14,9 +14,10 @@ final class TableViewController: UITableViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    tableView.register(cellType: MySimpleColorCell.self)
-    tableView.register(cellType: MyXIBTextCell.self)
-    tableView.register(cellType: MyXIBInfoCell.self)
+    /* Since we're using autoRegister there's no need to register these type */
+    //tableView.register(cellType: MySimpleColorCell.self)
+    //tableView.register(cellType: MyXIBTextCell.self)
+    //tableView.register(cellType: MyXIBInfoCell.self)
```

Comment:
For the sake of example and to show how to register cells manually with Reusable, maybe only auto-register one of the 3? Or keep one of the 3 manual?